docs/source/troubleshooting.rst

```diff
@@ -226,9 +226,32 @@ On Ubuntu / Debian:
 .. code-block:: bash
 
     /usr/lib/python2.7/dist-packages/st2common/bin/st2-self-check
-    
+
 On RedHat / Fedora:
 
 .. code-block:: bash
 
     /usr/lib/python2.7/site-packages/st2common/bin/st2-self-check
+
+
+Support
+-------
+
+Support is available to you via `email <support@stackstorm.com/>`_ or via
+`slack-community <https://stackstorm.typeform.com/to/K76GRP)/>`_.
+
```

Comment:
We stand behind |st2| and make the best effort to support you, fast. Ask for support at: 

* slack-community if you're not slacker good excuse to become one; you can search for similar problems and .. (_make a point it's persistent_)
* IRC - _ feel good about making a joke about hardcore linux hackers :)_
* support@stackstorm.com - this way we can get back to you